test/whitequark/test_hash_pair_value_omission_1.parse-tree-whitequark.exp

```diff
@@ -0,0 +1,4 @@
+s(:hash,
```

Comment:
this is not specific to your PR, but i noticed that all of our `.exp` tests have the form `test_name.rb.<pass>.exp` except for most of the `parse-tree-whitequark` tests, which don't have the `.rb` in the file name.

do you know why that is? while you're working with these files, would do you mind putting together a change to make that more consistent? (i'd do it, but i'd rather not step on your toes while you're doing this work)

Reply:
It's coming from the import script here https://github.com/sorbet/sorbet/blob/02191d20e3f7c341df69d0ce863fa70866aaad29/tools/scripts/import_whitequark.rb#L256

I can open an another PR to modify the script and rename everything under `test/whitequark`.